lib/linguist/languages.yml

```diff
@@ -5273,6 +5274,7 @@ YAML:
   - ".clang-format"
   - ".clang-tidy"
   - ".gemrc"
+  - "glide.lock"
```

Comment:
I'm clearly nitpicking here, but the double quotes are not required when the name doesn't start with a dot (we try to follow Ruby's YAML format).